src/components/admin/article/components/ImageField.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { ValidatedImage } from 'components/admin/form-components/validated-fields';
+
+const ImageUrlField = (props) => {
```

Comment:
Oh you did change the name here ;) (damn default imports!). Then just propagate it across the rest of the files, and maybe also change the ValidatedImage to be ValidatedImageUrl and the other relevant places (updateImage etc.)

Reply:
Oops you commented this to.

part of rename ImageUrlField - TODO